src/java/picard/sam/MergeSamFiles.java

```diff
@@ -52,7 +52,17 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Merges multiple SAM/BAM files into one file.",
+        usage = "Merges multiple SAM/BAM files into one file." +
```

Comment:
Might be useful to note that care should be taken to only merge if the read groups are specified appropriately.